stac_fastapi/types/stac_fastapi/types/rfc3339.py

```diff
@@ -21,6 +20,34 @@
 ]
 
 
+# Borrowed from pystac - https://github.com/stac-utils/pystac/blob/main/pystac/utils.py
```

Comment:
@jonhealy1 this borrow is fine by me, let's use a permalink though.

```suggestion
# Borrowed from pystac - https://github.com/stac-utils/pystac/blob/f5e4cf4a29b62e9ef675d4a4dac7977b09f53c8f/pystac/utils.py#L370-L394
```

Reply:
Thanks @gadomski, a permalink is much better